src/test/java/pl/teksusik/kick4j/events/KicksGiftedEventTest.java

```diff
@@ -0,0 +1,79 @@
+package pl.teksusik.kick4j.events;
+
+import java.time.Instant;
+import org.junit.jupiter.api.Test;
+import pl.teksusik.kick4j.events.type.ChannelSubscriptionCreatedEvent;
+import pl.teksusik.kick4j.events.type.EventUser;
+import pl.teksusik.kick4j.events.type.Gift;
+import pl.teksusik.kick4j.events.type.KicksGiftedEvent;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class KicksGiftedEventTest extends KickEventDispatcherTest {
+    private static final String PAYLOAD = """
```

Comment:
Please update payload to match that one provided in [KickDev documentation](https://docs.kick.com/events/event-types#kicks-gifted)